distribution/docker/src/docker/Dockerfile

```diff
@@ -16,7 +16,9 @@ FROM centos:7 AS builder
 ENV PATH /usr/share/elasticsearch/bin:$PATH
 ENV JAVA_HOME /opt/jdk-${jdkVersion}
 
-RUN curl -L --retry 8 -s ${jdkUrl} | tar -C /opt -zxf -
+RUN curl -L --retry 8 -s ${jdkUrl} --continue-at - --output /tmp/jdk.tar.gz && \
```

Comment:
I think it might be worthwhile to use `-S` here so we can see the errors. Is there significant motivation for suppressing _all_ output here? AFAIK this doesn't affect the image in any way.